src/couch/src/couch_server.erl

```diff
@@ -416,10 +416,11 @@ all_databases(Fun, Acc0) ->
                 true,
                 fun(Filename, AccIn) ->
                     NormFilename = couch_util:normpath(Filename),
-                    case NormFilename -- NormRoot of
-                        [$/ | RelativeFilename] -> ok;
-                        RelativeFilename -> ok
-                    end,
+                    RelativeFilename =
+                        case NormFilename -- NormRoot of
+                            [$/ | Filename] -> Filename;
```

Comment:
We don't want to use `Filename` as that will have to match with already bound `Filename` from `fun(Filename, AccIn)`. Inside case clauses, we can just use a shorted variable if there is minimal chance collision, so something like `FN` perhaps?

Reply:
Ohhhh, I totally overlooked that! Of course, that won't work! Will fix this!